manifests/base/image.yaml

```diff
@@ -3,7 +3,7 @@ kind: Qserv
 metadata:
   name: qserv
 spec:
-  image: qserv/lite-qserv:2022.1.1-rc1-50-gd066e4e64
+  image: qserv/lite-qserv:2022.2.1-rc1-52-g7ebaae6fc
```

Comment:
Don't you want to switch to the latest stable version of Qserv?
```
qserv/lite-qserv:2022.4.2-rc1
```
It would be certainly needed to implement the readiness/liveliness probes of the workers Registry based on this service:
```
/meta/version
```

Reply:
Nevermind! I noticed this was fixed in a later commit.